### PR TITLE
Typo fix in test_eval_arb

### PR DESCRIPTION
**Pull request by abhinavagarwal07**

### CODE REVIEW

File: symengine/tests/eval/test_eval_arb.cpp [hunk 1]
```diff
@@ -625,7 +625,7 @@ TEST_CASE("Cosh: eval_arb", "[eval_arb]")
     arb_clear(a);
 }
 
-TEST_CASE(", eval_arb", "[eval_arb]")
+TEST_CASE("Tanh, eval_arb", "[eval_arb]")
```
Comment (Sumith1896) [inline symengine/tests/eval/test_eval_arb.cpp]: `"Tanh: eval_arb"`

**Comment by Sumith1896** [comment]
+1 apart from my comment. Thanks!

**Comment by Sumith1896** [comment]
Can you squash with one commit message? 
Not necessary for merging this but doing so is a lot cleaner. Let me know if you want to do so, else we can merge it directly.

**Comment by abhinavagarwal07** [comment]
@Sumith1896 
squashed commits.
Thanks for suggestion

**Comment by Sumith1896** [comment]
Good to merge! Thanks @abhinavagarwal07

**Comment by isuruf** [comment]
Thanks @abhinavagarwal07

**Pull request merged**